MetaMask/metamask-extension
# chore: Update and use selectors for which chains to poll

## **Description**


Updates polling hooks to use selectors indicating which chains to poll.

Also update the selectors to only return the current chain when portfolio view is filtered to the current chain.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28586?quickstart=1)

## **Related issues**


## **Manual testing steps**

With PORTFOLIO_VIEW=1

1. When filtering all chains, should see requests in network tab hitting all chains
2. When filtering for current chain, should see requests in network tab hitting only the current chain

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
